### ISSUE TITLE
question about java bindings (unicorn java packaging)

### ISSUE THREAD

**Issue by blacktop** [unicorn-engine/unicorn#854]
I have a noob question about the Java bindings

I want to statically include the `libunicorn` deps into a java application so I can just ship the java app? 

I am new to java can anyone help? I have tried bundling with eclipse, but having trouble
when I include the `libunicorn_java.so` and `libunicorn.so` and `libunicorn.a` file it doesn't know about the BlockHook class etc etc

It seems to need to `libunicorn.so` files to be in the `/usr/lib` directory?

**Comment by blacktop** [comment]
Have you packaged it before @cseagle ?

**Comment by cseagle** [comment]
I have not packaged it up before. Are you including unicorn.jar in your project? It contais BlockHook.class and all of the other class files.

**Comment by blacktop** [comment]
@cseagle I apologize for the dumb question.  All I had to do was statically compile unicorn and then the JAR used that to create the much larger statically complied `libunicorn_java.so` and now it works.  I was getting tripped up because I had to remove all the other times I tried i.e. the `unicorn.jar/libunicorn_java.so` in`java/jre/lib/ext` and the `libunicorn.so|a` in `/usr/lib` and then my java app would only look in it's own `lib` folder etc etc.

Thank you for the response sir!
